chainerx_cc/chainerx/routines/n_step_rnn.cc

```diff
@@ -153,8 +153,8 @@ std::vector<std::vector<Array>> OneDirectionalLoop(
         *c = Reshape(*c, h_shape);
     }
     std::vector<Array> h_list;
-    for (size_t i = 0; i < xs.size(); i++) {
-        Array x_t = xs[i];
+    for (auto& x : xs) {
+        Array x_t = x;
```

Comment:
You can just remove `x_t` and use `x` instead?

Reply:
done